austrakka/components/sequence/funcs.py

```diff
@@ -615,26 +615,23 @@ def _create_samples(
         owner_org: str,
         shared_groups: list[str],
 ) -> None:
-    with tempfile.NamedTemporaryFile(suffix=".csv") as tmp:
-        csv_str = StringIO()
-        rows = [[METADATA_FIELD_SEQ_ID]]
-        for seq_id in seq_ids:
-            logger.info(f"Creating sample with {METADATA_FIELD_SEQ_ID} {seq_id} if required")
-            rows.append([seq_id])
-        csv.writer(csv_str).writerows(rows)
-        csv_str.seek(0)
-        with open(tmp.name, 'w', encoding='utf8') as file:
-            shutil.copyfileobj(csv_str, file)
-
-        with open(tmp.name, 'rb') as file:
-            file.seek(0)
-            add_metadata(
-                file, 
-                owner_org, 
-                shared_groups, 
-                "min", 
-                blanks_will_delete=False,
-                batch_size=5000)
+    for seq_id in seq_ids:
+        try:
+            api_post(f'{SUBMISSION_PATH}/Sample', data={
+                'name': seq_id,
+                'owner': owner_org,
+            })
+        except FailedResponseException as ex:
+            sample_exists_msg = f"Sample {seq_id} already exists"
+            if any(sample_exists_msg in m for m in ex.get_error_messages()):
+                logger.warning(sample_exists_msg)
+            else:
+                raise ex
+    for group in shared_groups:
+        api_patch(f'{SAMPLE_PATH}/Share', data={
+            'seqIds': seq_ids,
+            'groupName': f'{group}-Group',
```

Comment:
@samcarswell is this correct? Appending "-Group" is only necessary if starting from the project abbreviation; the group name will already have it